Medicina/Misc/MedScripts/src/med-ssh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+
+print_help() {
+    echo " "
+    echo "`basename $0` v1.0"
+    echo " "
+    echo "Log project into the DISCOS system homes."
+    echo "Usage: '`basename $0` [-p|--port portnumber] <projectid>@<gatein>'" 
+    echo "       '`basename $0` -h|--help'"
+    echo "-h | --help prints this help message and exit"
+    echo "-p | --port ssh port, if different from the default one"
+    echo "       <projectid> indicates the project you wanto to log in."
+    echo "       <gatein> access point into the observatory lan. This information is provided by local staff."
+    echo "                Use 'local' if already inside the local area network."	     
+    exit 1
+}
+
+current_system="LINUX"
+ssh_port="22"
+project=""
+gatein=""
+local="FALSE"
+
+while [[ $# -gt 0 ]]
+do
+key="$1"
+case $key in
+    -h|--help)
+    print_help
+    ;;
+    -p|--port)
+    ssh_port=$2
+    shift
+    shift
+    ;;
+    if [[ $1 == *"@"* ]]; then
```

Comment:
Somehow I'm getting this error on Ubuntu 20.04 LTS
`./med-ssh: line 36: syntax error near unexpected token '[['`
`./med-ssh: line 36: '    if [[ $1 == *"@"* ]]; then'`